project_sequence/views/project_project.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Moduon Team S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0) -->
+<data>
+
+<record id="project_sequence_form_edit_project" model="ir.ui.view">
+    <field name="name">Project_sequence_project_edit</field>
+    <field name="model">project.project</field>
+    <field name="inherit_id" ref="project.edit_project" />
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='partner_id']" position="after">
```

Comment:
Please pay attention to the module requirements. The field goes into the "settings" tab.

Reply:
Hi, the partner_id is in settings tab.